Source/MMRecord/MMRecord.h

```diff
@@ -204,6 +204,66 @@ extern NSString * const MMRecordAttributeAlternateNameKey;
  callback queues will be added in a future version.
  */
 
+///-----------------------------------------------
+/// @name Block typedefs for MMRecordOptions
+///-----------------------------------------------
+
```

Comment:
Had to move the typedefs up earlier in the file, so they can be used in the properties for MMRecord.